src/components/Login/Login.js

```diff
@@ -6,16 +6,34 @@ import {
   ButtonLogin,
   SpotifyButton,
   Backward} from "../../styled/Utils";
-
+import querystring from 'querystring';
 
 const Login = () => {
+  
+  const goToLogin = () => {
+    let state = 'some-state-of-my-choice';
```

Comment:
This should be a random string that's not visible (like clientId) on release version.

https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.1

Reply:
yes, for testing we use things like this for testing and development, then with react context we set the state and use it for checking